detect_secrets/core/audit.py

```diff
@@ -66,6 +70,45 @@ def audit_baseline(baseline_filename):
     _save_baseline_to_file(baseline_filename, original_baseline)
 
 
+def _get_baseline_from_file(filename):  # pragma: no cover
```

Comment:
Raised you one further and re-org'd them based on the order that they are called.

Reply:
<3